From baff3d131686ad5783ed1e1d841a0021cc537962 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Thu, 1 Jul 2021 11:21:35 +0200
Subject: [PATCH] mmc: core: Disable CQE

On the customer board we get a lot of

mmc2: running CQE recovery

So let's disable CQE for now.

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Florin Sarbu <florin@balena.io>
---
 drivers/mmc/core/mmc.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/mmc/core/mmc.c b/drivers/mmc/core/mmc.c
index d805f8450719..2e3949728351 100644
--- a/drivers/mmc/core/mmc.c
+++ b/drivers/mmc/core/mmc.c
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: GPL-2.0-only
+
 /*
  *  linux/drivers/mmc/core/mmc.c
  *
@@ -1864,7 +1864,7 @@ static int mmc_init_card(struct mmc_host *host, u32 ocr,
 	 * be used with Command Queue.
 	 */
 	card->ext_csd.cmdq_en = false;
-	if (card->ext_csd.cmdq_support && host->caps2 & MMC_CAP2_CQE) {
+	if (0 ) {
 		err = mmc_cmdq_enable(card);
 		if (err && err != -EBADMSG)
 			goto free_card;
@@ -1875,6 +1875,7 @@ static int mmc_init_card(struct mmc_host *host, u32 ocr,
 			card->ext_csd.cmdq_depth = 0;
 		}
 	}
+
 	/*
 	 * In some cases (e.g. RPMB or mmc_test), the Command Queue must be
 	 * disabled for a time, so a flag is needed to indicate to re-enable the
-- 
2.37.2

